### PR TITLE
Llama3.3-70b-Galaxy - Add determinism seed impl for prefill+decode 

### PR DESCRIPTION
**Pull request by djordje-tt**
### Ticket
https://github.com/tenstorrent/tt-metal/issues/35017

### Problem description
Check ticket above and it's parent for description.

### What's changed
There is host workaround where we keep RNG state on host and get next rnd values before each sampling iteration. Once we get next rnd values, we push a tensor to device and use that tensor to call ttnn.manual_seed(seeds) that is called just before ttnn.sampling. This way nothing will affect PRNG state and we are passing each iteration next rnd value generated on host so it's guarantee for determinism.

### CODE REVIEW

File: models/demos/llama3_70b_galaxy/tt/generator.py [hunk 1]
```diff
@@ -83,6 +83,7 @@ def warmup_prefill_traces(
         empty_slots=None,
         tt_out_logits_all_users=None,
     ):
+        return
```
Comment (rdraskicTT) [inline models/demos/llama3_70b_galaxy/tt/generator.py:98]: To be removed before merging

**Review by yieldthought — APPROVED** [review]
